package-lock.json

```diff
@@ -22352,12 +22352,6 @@
         "safe-buffer": "^5.1.1"
       }
     },
-    "node-forge": {
```

Comment:
node-forge was a top level dependency before. selfsigned has gone up by a patch increment and gained the dependency `node-forge` which is fine,  but i'm trying to work out how this can now be removed from here?

Reply:
@thatguynamedandy  was that `selfsigned` was previously using this top-level `node-forge` install, but the newer version/best practice requires it to be its own specified install?

Reply:
I'll be honest I'm not entirely sure why npm moved node-forge down a layer in the dependency tree, I've checked through and there is no reference to the old one. I think I'll go ahead and merge but keep an eye on it.

Reply:
cool, if there's no reference seems good to go :-)